### PR TITLE
feat(frontend): add Claude commands and update README

### PR DESCRIPTION
**Pull request by talissoncosta**
## Summary
- Add Claude commands for common development workflows
- Update README with unit testing documentation
- Include example test demonstrating the pattern
- Builds on top of #6486 (frontend developer documentation)

### New Claude commands
- `frontend/.claude/commands/unit-test.md` - Analyse files and generate unit tests following project patterns
- `frontend/.claude/commands/pr-review.md` - Review GitHub PRs with structured feedback
- `frontend/.claude/commands/find-usages.md` - Find all usages of a component/function for impact analysis

### Documentation
- `frontend/README.md` - Added unit testing section (how to run unit tests, filter by file)

### Example
- `frontend/common/utils/__tests__/isFreeEmailDomain.test.ts` - Example test generated using the `/unit-test` command

## How `/unit-test` works
The command instructs Claude to:
1. Check if tests already exist for the file
2. Analyse the source file for exported functions
3. Generate tests following the pattern in `format.test.ts` (Jest + `it.each`)
4. Include edge cases (null, undefined, empty values)
5. Run the tests to verify

## Test plan
- [x] Generated test for `isFreeEmailDomain.ts` - 12 tests pass
- [x] README unit test commands work correctly

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: frontend/.claude/commands/find-usages.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
```
Comment (Zaimwa9) [inline frontend/.claude/commands/find-usages.md:1]: really nice one

**Review by Zaimwa9 — APPROVED** [review]
Yeah looks good, I really liked the find-usages
